pennylane/pauli/pauli_arithmetic.py

```diff
@@ -540,9 +540,9 @@ class PauliSentence(dict):
     **Examples**
 
     >>> ps = PauliSentence({
-            PauliWord({0:'X', 1:'Y'}): 1.23,
-            PauliWord({2:'Z', 0:'Y'}): -0.45j
-        })
+    ...     PauliWord({0:'X', 1:'Y'}): 1.23,
+    ...     PauliWord({2:'Z', 0:'Y'}): -0.45j,
```

Comment:
Do we need the comma here? Code works either way
```suggestion
    ...     PauliWord({2:'Z', 0:'Y'}): -0.45j
```